resources/keywords/create_provider_account.resource

```diff
@@ -26,6 +27,7 @@ User Navigates To Create Provider Account Screen From Database Access Page
     ...    Select Database Provider Account From Create Button DropDown
     ...    ELSE
     ...    Click Element    ${btnCreateProviderAcc}
+    Sleep    8s         #Blind sleep to mitigate false failures due to auto page refresh
```

Comment:
Instead of hard wait, use "Wait Until Keyword Succeeds" assigned with target element